AutomaticPackages.md

```diff
@@ -36,30 +34,18 @@ There are currently two methods that can be used to maintain automatic packages:
 
 ## Automatic Updater (AU)
 
-You can learn more about the AU updater via its [documentation](https://github.com/majkinetor/au/blob/master/README.md). It is a PowerShell module, so you are doing everything with PowerShell scripts. You can also run everything on AppVeyor so you are not required to schedule updater on your own machine.
+The [Chocolatey Automatic Package Updater Module](https://github.com/majkinetor/au/blob/master/README.md) is a PowerShell module which implements functions that can be used to automate Chocolatey package updates.
```

Comment:
Can we use 'AU Module' here instead of the 'Chocolatey Automatic Package Updater Module'. As the project isn't a Chocolatey project then using the Chocolatey 'branding' may confuse people. The same reason applies for packages in the Community Repository.

Reply:
I understand. However, I think _AU Module_ alone doesn't say much about what it is and could potentially confuse a person reading this for the first time. How about _Automatic Updater Module_?

Reply:
I suggested 'AU Module' as it's really known as 'AU' but what you have suggested is fine.